analyses/NJ_cd_2000/03_sim_NJ_cd_2000.R

```diff
@@ -0,0 +1,32 @@
+###############################################################################
+# Simulate plans for `NJ_cd_2000`
+# © ALARM Project, July 2025
+###############################################################################
+
+# Run the simulation -----
+cli_process_start("Running simulations for {.pkg NJ_cd_2000}")
+
+set.seed(2000)
+plans <- redist_smc(map, nsims = 4e3, runs = 10, counties = pseudo_county)
+
+plans <- plans %>%
+  group_by(chain) %>%
+  filter(as.integer(draw) < min(as.integer(draw)) + 500) %>% # thin samples
+  ungroup()
+plans <- match_numbers(plans, "cd_2000")
+
+cli_process_done()
+cli_process_start("Saving {.cls redist_plans} object")
+
```

Comment:
@KT0416838 We're on a meeting right now, but we could add the contiguity check that we are discussing with @christopherkenny here.